website_snippet_breadcrumb/views/snippets.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <data>
+
+        <template id="snippet_selection" inherit_id="website.snippets">
+            <xpath expr="//div[@id='snippet_content']" position="inside">
+                <div>
+                    <div class="oe_snippet_thumbnail">
+                        <img src="/website_snippet_breadcrumb/static/src/img/snippets/s_breadcrumb.png"
+                             class="oe_snippet_thumbnail_img"/>
+                        <span class="oe_snippet_thumbnail_title">
+                            Breadcrumb
+                        </span>
+                    </div>
+                    <section class="oe_snippet_body website_snippet_breadcrumb" data-revert-order='true' data-include-self='false'>
+                        <div class="parametricTemplate">
+                          <p>Dynamic Breadcrumb: The content will be replaced after saving the page</p>
```

Comment:
I think a `ol.breadcrumb>li` placeholder would be more user friendly than a normal paragraph, to have a better preview of how will it look like.